packages/fulfillment/src/models/fullfilment-set.ts

```diff
@@ -46,11 +60,9 @@ export default class FulfillmentSet {
   @Property({ columnType: "jsonb", nullable: true })
   metadata: Record<string, unknown> | null = null
 
-  @ManyToMany(() => ServiceZone, "fulfillment_sets", {
-    owner: true,
-    pivotTable: "fulfillment_set_service_zones",
-    joinColumn: "fulfillment_set_id",
-    inverseJoinColumn: "service_zone_id",
+  @OneToMany(() => ServiceZone, "fulfillment_set", {
+    cascade: [Cascade.PERSIST, "soft-remove"] as any,
+    orphanRemoval: true,
```

Comment:
question: I thought we didn't use this because of its "aggressiveness". I am fine to keep it in, but just wanted to point it out in case it was an oversight

Reply:
Here it a requirements. Because since i am bulk deleting the entities, then when mikro orm manage the collection under the hood it will complain that thos configuration is missing. Also, i am in a case where if the fulfillment set is removed completely, then we can have orphan service zones since the foreign key is non nullable. Same goes between service zones and geo zones